### PR TITLE
[test] Remove duplicated tests

### PR DESCRIPTION
**Pull request by Honry**
- br.wast: "$type-operand-missing" and "$type-operand-missing-in-block"
   are similar to "$type-arg-empty-vs-num"
 - return.wast "$type-operand-missing" is similar to "$type-value-empty-vs-num"
 - if.wast: no duplicates

### CODE REVIEW

**Comment by Honry** [comment]
@rossberg, PTAL. Thanks!

File: test/core/br.wast [hunk 1]
```diff
@@ -442,25 +442,6 @@
...
-      (i32.const 0)
-      (block (result i32) (br 0))
-      (i32.eqz) (drop)
-    )
-  )
-  "type mismatch"
-)
 (assert_invalid
   (module
     (func $type-operand-missing-in-if
```
Comment (rossberg) [inline test/core/br.wast]: I believe the this and the next are duplicates  of `$type-{then,else}-break-empty-vs-num` in if.wast, so can be removed as well.

File: test/core/if.wast [hunk 2]
```diff
@@ -768,8 +768,6 @@
   "type mismatch"
 )
 
-;; TODO: Compare above "*operand-missing*" tests to others, identify and remove duplicates.
```
Comment (rossberg) [inline test/core/if.wast]: Can we rename the above to `$type-condition-empty-*`, to closer match the naming scheme of the other tests?

Also, `in-if` could be `in-then` for symmetry.

File: test/core/return.wast [hunk 3]
```diff
@@ -320,14 +320,6 @@
...
-  (module
-    (func $type-operand-missing (result i32)
-      (return)
-    )
-  )
-  "type mismatch"
-)
 (assert_invalid
   (module
     (func $type-operand-missing-in-block (result i32)
```
Comment (rossberg) [inline test/core/return.wast]: Can we rename these to $type-value-empty-vs-num-in-*` and move them next to the basic version of the test?

File: test/core/br.wast (hunk 1 above)
Comment (Honry) [reply test/core/br.wast]: Indeed, I will remove them as well.

File: test/core/if.wast (hunk 2 above)
Comment (Honry) [reply test/core/if.wast]: Yeh, good suggestion, I will rename them soon.

File: test/core/return.wast (hunk 3 above)
Comment (Honry) [reply test/core/return.wast]: Fine. :)

**Comment by Honry** [comment]
@rossberg, many thanks for your review, all comments have been addressed. Pls. take a look again. :)
